backend/lib/robots/roborock/RoborockQuirkFactory.js

```diff
@@ -14,6 +14,62 @@ class RoborockQuirkFactory {
      */
     getQuirk(id) {
         switch (id) {
+            case RoborockQuirkFactory.KNOWN_QUIRKS.AUTO_EMPTY_LENGTH:
+                return new Quirk({
+                    id: id,
+                    title: "Auto Empty Duration",
+                    description: "Set the dustbin emptying duration when triggered by robot after docking.",
+                    options: ["Smart", "Quick", "Daily", "Strong", "Max"],
```

Comment:
As `strong` is omitted in the app according to https://github.com/rytilahti/python-miio/issues/1107#issuecomment-893867891, I'd probably also omit it in Valetudo as there could be a hardware reason for that.
Maybe it can cause trouble due to overloaded motors or similar

Reply:
Good point. I thought that it wouldn't hurt to add it, but you're right, maybe they had a point in excluding it...